backend/grant/milestone/models.py

```diff
@@ -72,14 +75,56 @@ def make(milestones_data, proposal):
                 m = Milestone(
                     title=milestone_data["title"][:255],
                     content=milestone_data["content"][:255],
-                    date_estimated=datetime.datetime.fromtimestamp(milestone_data["date_estimated"]),
+                    days_estimated=str(milestone_data["days_estimated"])[:255],
                     payout_percent=str(milestone_data["payout_percent"])[:255],
                     immediate_payout=milestone_data["immediate_payout"],
                     proposal_id=proposal.id,
                     index=i
                 )
                 db.session.add(m)
 
+    @staticmethod
+    def set_v2_date_estimates(proposal):
```

Comment:
The `MilestoneDeadline` task tests already involve this method, but it might be worth it to write some tests for this method directly.

Reply:
Added additional tests.